build-release-tools/application/jump_version.py

```diff
@@ -83,24 +108,23 @@ def update_changelog(self, message=None):
         if not debian_exist:
             # Handle repository which contains debianstatic/repository_name folder, 
             # for example: debianstatic/on-http
-            for filename in os.listdir(self._repo_dir):
-                if filename == "debianstatic":
-                    debianstatic_dir = os.path.join(self._repo_dir, "debianstatic")
-                    for debianstatic_filename in os.listdir(debianstatic_dir):
-                        if debianstatic_filename == repo_name:
-                            debianstatic_repo_dir = "debianstatic/{0}".format(repo_name)
-                            common.link_dir(debianstatic_repo_dir, "debian", self._repo_dir)
-                            linked = True
+            linked = self.link_debianstatic()
 
         if not debian_exist and not linked:
-            return False
+            return
 
-        print "start to update changelog of {0}".format(self._repo_dir)
         # -v: Add a new changelog entry with version number specified
         # -b: Force a version to be less than the current one
         # -m: Don't change (maintain) the trailer line in the changelog entry; i.e.
         #     maintain the maintainer and date/time details
         try:
+            version = self.get_current_version()
+            # TBD: check whether the commit in manifest is the commit to jump version
+            if version == self._version:
+                print "The version of {0} is already {1}".format(repo_name, self._version)
```

Comment:
the log maybe 'the version of xx is already xx, don't need start to update changelog' ? compared with following codes